File: Programmers/Lv1/Lv1_이상한문자만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
	string answer = "";
	int index = 0;

	for (int i = 0; i < s.length(); i++) {
		if (s[i] != ' ') {
			if (index % 2 == 1)
				s[i] = tolower(s[i]);
			else
				s[i] = toupper(s[i]);
			index++;
		}
		else
			index = 0;
	}

	return s;
}

File: Programmers/Lv1/Lv1_이상한문자만들기.py
def solution(s):
    answer = ''
    index = 0
    for i in s:
        if i != " ":
            if index%2==0:
                answer += i.upper()
            else:
                answer += i.lower()
            index += 1
        else:
            index = 0
            answer += i
    return answer